### DISCUSSION TITLE
action 调试开启后也没看到调试界面，找不到调试入口

### DISCUSSION THREAD

**Discussion by jacklp1** [XiaoMi/ha_xiaomi_home#50] [Q&A]
action 调试开启后也没看到调试界面，找不到调试入口

**Comment by JimmyKmi** [comment]
action 调试是在生成实体列表时额外新增可执行的action实体方便测试，如需要使用虚拟事件参考：https://note.p.lzj.ac.cn/s/bd635048-ae38-43c9-9f31-d61175a01cf6/doc/5lit5p6i572r5ywz6jma5ouf5lql5lu26kem5yr5yk5omn6kgm-BBcVquoizJ
